elixir/apps/web/lib/web/live/groups/show.ex

```diff
@@ -5,13 +5,7 @@ defmodule Web.Groups.Show do
 
   def mount(%{"id" => id}, _session, socket) do
     with {:ok, group} <-
-           Actors.fetch_group_by_id(id, socket.assigns.subject,
-             preload: [
-               provider: [],
-               created_by_identity: [:actor],
-               created_by_actor: []
-             ]
-           ) do
+           Actors.fetch_group_by_id(id, socket.assigns.subject, preload: [provider: []]) do
```

Comment:
Can you just do?

```suggestion
           Actors.fetch_group_by_id(id, socket.assigns.subject, preload: :provider) do
```

Reply:
Ah, yeah, good catch.  I was too focused on the `created_by_` stuff that I didn't really look at what was left.  I'll get it updated.